cfme/infrastructure/provider/__init__.py

```diff
@@ -167,7 +167,7 @@ def vm_name(self):
         return version.pick({
             version.LOWEST: "VMs",
             '5.5': "VMs and Instances",
-            '5.7': "Virtual Machines"})
+            '5.7.1': "Virtual Machines"})
```

Comment:
Noted, your 5.7 vs 5.7.1 change makes sense now.

We've been removing any version picking prior to 5.6 for some time now.  If we need to test against 5.5 we can do so on an older version of the repository.  In which case this would move 'VMs and Instances' to version.LOWEST which would cover 5.5, 5.6, and 5.7.0